### PR TITLE
Prevent systemd starting hotspot at boot

### PR DESCRIPTION
**Pull request by xipmix**
This addresses an issue noticed in #288, that we want hotspot.service to be controlled by wireless.js alone. Allowing systemd to start hotspot.service independently just makes things harder to debug,

### CODE REVIEW

File: volumio/bin/firststart.sh [hunk 1]
```diff
@@ -15,6 +15,11 @@ rm -v /etc/ssh/ssh_host_*
 echo "Generating SSH host keys"
 dpkg-reconfigure openssh-server
 
+echo "Ensuring hotspot startup is controlled only by wireless.js"
```
Comment (volumio) [inline volumio/bin/firststart.sh]: This should be removed, and we should not enable those services in the build script

Comment (xipmix) [reply volumio/bin/firststart.sh]: Thanks for that. I found a better place to do this, not sure if the firstboot.sh change is needed at all now.

**Comment by volumio** [comment]
Can you also remove the new lines on firststart.sh?

**Comment by xipmix** [comment]
Sure, done now. I have tested the disabling works on an existing system but I have not done a full build and tested that.

**Comment by xipmix** [comment]
Cleaned up and rebased.

**Comment by xipmix** [comment]
I built an image to test with this patch applied. At firstboot, hotspot runs (as expected). Then I disabled it in the UI and rebooted; system came up without hotspot. Then I enabled it in the UI and it started, finally disabled again and it stopped.

**Pull request merged**